setup.py

```diff
@@ -23,6 +27,30 @@
 
 ez_setup.use_setuptools()
 
+# 3-clause BSD: https://github.com/yt-project/yt/blob/master/COPYING.txt
+def test_openmp():
+  
+    tmpdir = tempfile.mkdtemp()
+    curdir = os.getcwd()
+    os.chdir(tmpdir)
+
+    compiler = os.environ.get('CC', distutils.sysconfig.get_config_var('CC')).split()[0]
```

Comment:
Maybe we no need "fallback" here (with `openmpi`), because
- We provide wheels for users (and we can use `openmpi` when we build wheels)
- If the user wants to compile itself, he should have **compiler** + **python headers (dev package)** -> what's a problem to install one more package (**openmpi**)?
- This looks like not really useful complication (I don't see the case when this is really needed)

CC: @piskvorky thoughts?